test/units/modules/network/check_point/test_cp_mgmt_network.py

```diff
@@ -46,13 +46,6 @@
 class TestCheckpointNetwork(object):
```

Comment:
On this file, there are some unused imports, unrelated to the commit but please take care of it as well.
`test/units/modules/network/check_point/test_cp_mgmt_network.py:22:1: F401 'units.modules.utils.AnsibleFailJson' imported but unused
from units.modules.utils import set_module_args, exit_json, fail_json, AnsibleFailJson, AnsibleExitJson
^
test/units/modules/network/check_point/test_cp_mgmt_network.py:25:1: F401 'ansible.module_utils.network.checkpoint.checkpoint.api_call' imported but unused
from ansible.module_utils.network.checkpoint.checkpoint import api_call
`